Lib/venv/scripts/nt/activate.bat

```diff
@@ -30,3 +39,7 @@ if defined _OLD_VIRTUAL_PATH (
 set "PATH=%VIRTUAL_ENV%\__VENV_BIN_NAME__;%PATH%"
 
 :END
+if defined _OLD_CODEPAGE (
+    "%SystemRoot%\System32\chcp.com" %_OLD_CODEPAGE% > nul
+    set _OLD_CODEPAGE=
```

Comment:
As you did above, there should be quotes around the assignment, just in case there's any white space at the end, e.g. `set "_OLD_CODEPAGE="`.